### ISSUE TITLE
ncm-dirperm: warning when running with 17.3

### ISSUE THREAD

**Issue by ned21** [quattor/configuration-modules-core#1137]
```
[INFO] running component: dirperm
---------------------------------------------------------
[WARN] Warning during loading of package NCM::Component::dirperm: Direct usage  of EDG::WP4::CCM::Element is deprecated and contains no functional code besides  a limited number of reexported functions and constants from other modules. at  /usr/lib/perl/EDG/WP4/CCM/Element.pm line 35.
```

$ rpm -q ncm-dirperm ccm
ncm-dirperm-17.3.0-1.noarch
ccm-17.3.0-1.noarch

**Comment by stdweird** [comment]
The compaitbiltiy layer scheduled for removal in https://github.com/quattor/CCM/issues/168https://github.com/quattor/CCM/issues/168 produces a warn, so it will fail 1.55 unittests once https://github.com/quattor/maven-tools/pull/162 is merged and 1.55 is released.

It typically requires little work, but needs to be done for all components.
If this is really annoying, tag this for 17.4, and we can do a sweep before the release. But i'd rather see a "sping cleanup" for all components...

**Comment by stdweird** [comment]
fixed in https://github.com/quattor/configuration-modules-core/commit/d0a2dae7f1db4441d8b3600bef80fdf3547f74da
